classes/Category.php

```diff
@@ -214,7 +214,8 @@ public function add($autoDate = true, $nullValues = false)
         if (!$this->doNotRegenerateNTree) {
             Category::regenerateEntireNtree();
         }
-        $this->updateGroup($this->groupBox);
+        // if access group is not set, initialize it with 3 default groups
+        $this->updateGroup(($this->groupBox === null)?$this->groupBox:[]);
```

Comment:
Your condition is wrong.
If groupBox is null, use groupBox, otherwise empty array.
So you never use groupBox if it's a proper array.

Reply:
oups, thanks